src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/ET1ServingControllerTest.java

```diff
@@ -0,0 +1,107 @@
+
+        @Test
+    void midServingDocumentOtherTypeNames() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(true);
+        when(ET1ServingService.generateOtherTypeDocumentName(anyList())).thenReturn("expectedDocumentName");
+        mvc.perform(post(SERVING_DOCUMENT_OTHER_TYPE_NAMES_URL)
+                        .content(jsonMapper.toJson(ccdRequest))
+                        .header("Authorization", AUTH_TOKEN)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.data.otherTypeDocumentName", notNullValue()))
+                .andExpect(jsonPath("$.errors", nullValue()))
+                .andExpect(jsonPath("$.warnings", nullValue()));
+        verify(ET1ServingService, times(1)).generateOtherTypeDocumentName(anyList());
+
+    }
+
+    @Test
+    void midServingDocumentRecipient() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(true);
+        when(ET1ServingService.generateEmailLinkToAcas(any())).thenReturn("expectedLink");
+        when(ET1ServingService.generateClaimantAndRespondentAddress(any())).thenReturn("expectedAddresses");
+
+        mvc.perform(post(SERVING_DOCUMENT_RECIPIENT_URL)
+                        .content(jsonMapper.toJson(ccdRequest))
+                        .header("Authorization", AUTH_TOKEN)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.data.claimantAndRespondentAddresses", notNullValue()))
+                .andExpect(jsonPath("$.data.emailLinkToAcas", notNullValue()))
+                .andExpect(jsonPath("$.errors", nullValue()))
+                .andExpect(jsonPath("$.warnings", nullValue()));
+        verify(ET1ServingService, times(1)).generateEmailLinkToAcas(any());
+        verify(ET1ServingService, times(1)).generateClaimantAndRespondentAddress(any());
+
+    }
+
+    @Test
+    void midServingDocumentOtherTypeNamesForbidden() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(false);
+        mvc.perform(post(SERVING_DOCUMENT_OTHER_TYPE_NAMES_URL)
+                        .content(jsonMapper.toJson(ccdRequest))
+                        .header("Authorization", AUTH_TOKEN)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isForbidden());
+    }
+
+    @Test
+    void midServingDocumentRecipientForbidden() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(false);
+        mvc.perform(post(SERVING_DOCUMENT_RECIPIENT_URL)
+                        .content(jsonMapper.toJson(ccdRequest))
+                        .header("Authorization", AUTH_TOKEN)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isForbidden());
+    }

```

Comment:
I'm not sure if something within these tests throws Exception, but if not, better not adding throws Exception unless you're forced to, otherwise it can mislead other developers.  Worth cleaning if cleaning is necessary on this